pyro/ops/tensor_utils.py

```diff
@@ -109,6 +109,39 @@ def periodic_cumsum(tensor, period, dim):
     return result
 
 
+def periodic_features(duration, max_period=None, min_period=None, **options):
+    r"""
+    Create periodic (sin,cos) features from max_period down to min_period.
+
+    This is useful in time series models where long uneven seasonality can be
+    treated via regression, and short seasonality can be treated via the
+    :func:`periodic_repeat` and :func:`periodic_cumsum` helpers. For example,
+    to combine yearly with weekly seasonality one could set
+    ``max_period=365.25`` and ``min_period=7``.
+
+    :param int duration: Number of discrete time steps.
+    :param float max_period: Optional max period, defaults to ``duration``.
+    :param float min_period: Optional min period (exclusive), defaults to 2 = Nyquist cutoff.
+    :param \*\*options: Tensor construction options, e.g. ``dtype`` and ``device``.
+    :returns: A ``(duration, 2 * ceil(max_period - min_period) - 2)``-shaped tensor of features
```

Comment:
nit: `2 * ceil(max_period - min_period) - 2` -> `2 * floor(max_period / min_period)`

Reply:
oops, will fix docs. but I'm pretty sure `ceil(x) - 1 != floor(x)` and that we need `ceil` here. Am I mistaken?

Reply:
nope, you are right. `arange(1, 3.)` returns `1, 2`

Reply:
fixed.